upload-api/functions/ucan-invocation-router.js

```diff
@@ -152,72 +155,127 @@ export async function ucanInvocationRouter(request) {
 
   const { UPLOAD_API_DID } = process.env
   const { PRIVATE_KEY, STRIPE_SECRET_KEY } = Config
-  const serviceSigner = getServiceSigner({ did: UPLOAD_API_DID, privateKey: PRIVATE_KEY })
+  const serviceSigner = getServiceSigner({
+    did: UPLOAD_API_DID,
+    privateKey: PRIVATE_KEY,
+  })
 
-  const tasksStorage = createTasksStorage(AWS_REGION, invocationBucketName, workflowBucketName)
-  const receiptsStorage = createReceiptsStorage(AWS_REGION, taskBucketName, invocationBucketName, workflowBucketName)
-  const allocationsStorage = createAllocationsStorage(AWS_REGION, allocationTableName, {
-    endpoint: dbEndpoint,
+  const agentStore = AgentStore.open({
+    store: {
+      connection: {
+        address: {
+          region: AWS_REGION,
+        },
+      },
+      buckets: {
+        message: { name: workflowBucketName },
+        index: { name: invocationBucketName },
+      },
+    },
+    stream: {
+      connection: { address: {} },
+      name: streamName,
+    },
   })
+
+  const allocationsStorage = createAllocationsStorage(
+    AWS_REGION,
+    allocationTableName,
+    {
+      endpoint: dbEndpoint,
+    }
+  )
   const blobsStorage = composeblobStoragesWithOrderedHas(
     createBlobsStorage(R2_REGION, carparkBucketName, {
       endpoint: carparkBucketEndpoint,
       credentials: {
         accessKeyId: carparkBucketAccessKeyId,
         secretAccessKey: carparkBucketSecretAccessKey,
-      }, 
+      },
     }),
-    createBlobsStorage(AWS_REGION, storeBucketName),
+    createBlobsStorage(AWS_REGION, storeBucketName)
   )
 
   const invocationBucket = createInvocationStore(
     AWS_REGION,
     invocationBucketName
   )
-  const taskBucket = createTaskStore(AWS_REGION, taskBucketName)
+
   const workflowBucket = createWorkflowStore(AWS_REGION, workflowBucketName)
-  const delegationBucket = createDelegationsStore(r2DelegationBucketEndpoint, r2DelegationBucketAccessKeyId, r2DelegationBucketSecretAccessKey, r2DelegationBucketName)
-  const subscriptionTable = createSubscriptionTable(AWS_REGION, subscriptionTableName, {
-    endpoint: dbEndpoint
-  });
+  const delegationBucket = createDelegationsStore(
+    r2DelegationBucketEndpoint,
+    r2DelegationBucketAccessKeyId,
+    r2DelegationBucketSecretAccessKey,
+    r2DelegationBucketName
+  )
+  const subscriptionTable = createSubscriptionTable(
```

Comment:
can we do any format changes in a different PR? This is already going to be quite big, and will make super difficult to review if all format is changing

Reply:
Yeah I'll undo the formatting changes, they were not my intention it's just vscode formatted everything with prettier because I forgot to disable it on this repo after deleting all my configs in attempt to solve the issue that was caused by docker images 😭